cookbook/build-admin-frontend.rst

```diff
@@ -34,7 +34,6 @@ Solution 2: Build manually with docker
    docker run --rm --interactive --tty --volume ${PWD}:/var/project node:14.16.0 /bin/bash
 
    # for completion: using another node version is possible by adjusting the tag of the node image
-   # docker run --rm --interactive --tty --volume ${PWD}:/var/project node:10.24.0 /bin/bash
```

Comment:
sulu is not compatible with node 10: https://github.com/sulu/skeleton/pull/109#discussion_r613282499